libs/util/src/lib/calculate-timeout-timer.ts

```diff
@@ -0,0 +1,20 @@
+import { AVERAGE_WPM } from '@razor/constants';
+
+const averageWPM = AVERAGE_WPM;
+
+/** Calculating timeout timer for the race text.
+ *
+ * @param {string} text - Race text.
+ * @returns {number} - Timeout timer in seconds.
```

Comment:
What this function does is to compute the timeout *duration*, in seconds.
_timer_ is a tool that could probably take the duration as input and start ticking down.
https://www.google.com/search?q=timer&rlz=1C5CHFA_enLK1008LK1008&sxsrf=ALiCzsbWZjHkVKmg-8WgAHID-QgS0NdxWw:1670048344717&source=lnms&tbm=isch&sa=X&ved=2ahUKEwi1seDy5tz7AhVsR2wGHQFUDi0Q_AUoAXoECAIQAw&biw=1680&bih=854&dpr=2#imgrc=FA4K7zfJ5cTN7M

Reply:
Change function name and file name 62831d4e600c6c062635c4cb3badab2db5ce7f61